pkg/client/integration_test.go

```diff
@@ -307,6 +308,7 @@ func RunAuthorizationCodeFlow(t *testing.T, opServer *httptest.Server, clientID,
 	}()
 	require.Less(t, capturedW.Code, 400, "token exchange response code")
 	// TODO: how to check the custom header was sent to the server?
+	// TODO: how to check the Autorization header was sent to the server? (AuthStyleInHeader)
```

Comment:
I wouldn't "care" about this in the scope of OIDC/rp. As this is in the oauth2 package scope to send the header, they should test for it. Or deal with a bug if it doesn't work as expected.

Reply:
Alright, I had in my mind that it could be test-worthy, if the option from `rp.WithAuthStyle` to the oauth2 library was passed or not.